### PR TITLE
feat: replace the current 'Admin Tools' tab (based on a flex-grid of …

### PR DESCRIPTION
**Pull request by drewwills**
…app launchers) with the <esco-content-grid> Web Component

admin-dashboard

##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

This is a very significant cosmetic improvement to the `Admin Tools` tab -- thanks @jgribonvald!

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by jgribonvald** [comment]
These request won't be a terrible effort ;)
It's easy to add a property to set the filter on a category and in this case why not showing directly the category name ? The title is an i18n, It may be great to extend the component by providing a custom i18n maybe.

on an other way do you need the filtering area ? It could be disabled by a property too, or automatically when a category is applied by default.
Also we can disable the title and the filtered area depending on a property.

**Comment by cousquer** [comment]
+1 for "uPortal Administration Tools"

**Comment by cousquer** [comment]
🤔 we need to find a newer set of icons... don't you think @ChristianMurphy ?

File: data/quickstart/portlet-definition/admin-dashboard.portlet-definition.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        <name>alternate</name>
+        <value>false</value>
+    </parameter>
+    <parameter>
+        <name>blockImpersonation</name>
+        <value>false</value>
+    </parameter>
+    <parameter>
+        <name>chromeStyle</name>
+        <value>no-chrome</value>
```
Comment (cousquer) [inline data/quickstart/portlet-definition/admin-dashboard.portlet-definition.xml]: 👍

**Comment by ChristianMurphy** [comment]
> thinking we need to find a newer set of icons... don't you think @ChristianMurphy ?

Newer icons could refresh the look of the admin console. :+1: 

A few ideas for sources of new icons:
* https://undraw.co/illustrations 
* https://fontawesome.com/icons?d=gallery&m=free
* https://materialdesignicons.com/

**Comment by jgribonvald** [comment]
I can share our set of icons, we planned to create a repository shared between some ESUP members but we can do it on up-contrib.

**Comment by cousquer** [comment]
> > thinking we need to find a newer set of icons... don't you think @ChristianMurphy ?
> 
> Newer icons could refresh the look of the admin console. 👍
> 
> A few ideas for sources of new icons:
> 
> * https://undraw.co/illustrations
> * https://fontawesome.com/icons?d=gallery&m=free
> * https://materialdesignicons.com/

undraw is impressive also... 👁

**Comment by drewwills** [comment]
This PR is now ready to merge (afaik).  I'm just waiting for the CI to finish.

reboot-admin-dashboard

**Comment by drewwills** [comment]
Thank you tremendously @jgribonvald!

**Comment by jgribonvald** [comment]
@drewwills It's a pleasure.

**Pull request merged**